www/nginx/src/opnsense/scripts/nginx/setup.php

```diff
@@ -296,7 +296,7 @@ function find_ca($refid)
 foreach ($nginx->errorpage->iterateItems() as $errorpage) {
     $uuid = str_replace('-', '', $errorpage->getAttributes()['uuid']);
     if (in_array($uuid, $used_errorpages)) {
-        $filename = "error_$uuid.html";
+        $filename = "error_$uuid.$errorpage->extension";
```

Comment:
```suggestion
        $filename = "error_{$uuid}.{$errorpage->extension}";
```

Reply:
I must believe you, I don't know php enough